content/docs/concepts/data_model.md

```diff
@@ -61,3 +58,14 @@ this:
     api_http_requests_total{method="POST", handler="/messages"}
 
 This is the same notation that [OpenTSDB](http://opentsdb.net/) uses.
+
+Names with the UTF-8 characters outside the recommended set must be escaped, using
```

Comment:
```suggestion
Names with UTF-8 characters outside the recommended set must be quoted, using
```

For completion you could also mention that quotes within the quotes must be escaped (e.g. `{"my metric name \"with quotes\""}`).

This also demonstrates the difference between escaping and quoting.

Reply:
Nice! Thanks, applying, but I would vote to keep it simple and not mention escaping quoting - hopefully not very common problem.